tools/specs-checker/download.go

```diff
@@ -50,7 +50,7 @@ func getAndSaveFile(specDocUrl, outFilePath string) error {
 	}()
 
 	// Download spec doc.
-	resp, err := http.Get(specDocUrl)
+	resp, err := http.Get(specDocUrl) /* #nosec G107 */
```

Comment:
Can we fix this? What is the solution for G107?

Reply:
G107: Url provided to HTTP request as taint input

Reply:
it doesn't want untrusted, dynamically provided URLs to perform http requests. The resolution is to hardcode the spec url via a constant, or at least the base path perhaps

Reply:
In this case, there is a base path specified in a function above and this function is unexported, so cannot be used maliciously by anyone else